### ISSUE TITLE
Upgrade to Angular 2.0

### ISSUE THREAD

**Issue by zeeshanjan82** [amcdnl/systemjs-test#37]
Are there any plans to upgrade this seed project to use Angularjs2.0 instead. Can you refer me to some place where I could find similar implementation using AngularJS2.0

**Comment by amcdnl** [comment]
Angular2 is still very prototype, I'm waiting for things to stabilize before doing too much.

**Comment by zeeshanjan82** [comment]
@amcdnl I think its the right time to move forward with Angular2. What do you think? I am thinking of the seed project very similar to this one but this time it should include React.js and Angular2.

**Comment by amcdnl** [comment]
My main project is already using ng1 + ng2. We plan to upgrade this soon-ish but it will probably be a separate repo.

**Comment by zeeshanjan82** [comment]
@amcdnl I can work on this, I am currently looking for upgrading my knowledge on angular2 and after learning what in angular2 I would like to apply my knowledge and understanding on creating a seed project with you. Please let me know how can I contribute in creating a seed project for using angular2

**Comment by amcdnl** [comment]
@zeeshanjan82 Absolutely! I have gotten in working in my internal app with Babel. Here is the gist if you wanna take a stab at it! https://gist.github.com/amcdnl/baeb5fd5f6ad7aa76c4d897cbe860e86

**Comment by zeeshanjan82** [comment]
thanks @amcdnl I really appreciate. Will go over the gist

**Comment by zeeshanjan82** [comment]
@amcdnl after doing some more research I have realized that many things have changed in angular2, now we have typescript which has some real advantages over using babel and es6. Also angular2 now internally uses system.js and does lazy loading. Considering these things I think a lot is going to change in the seed project which is build for angular2. What is your opinion on this?

**Comment by amcdnl** [comment]
@zeeshanjan82 - 
- I am shying away from TS at the moment
- It does NOT do lazy loading YET, I'm sticking w/ ui-router ( it now supports ng2 too )
- This project uses SystemJS, not sure what you mean there
- The only major major change would be the build system w/ the CLI WHEN that drops, seemed pretty far off when I was at ngconf last wk

**Comment by zeeshanjan82** [comment]
Thanks @amcdnl what is your openion on this seed project, I need the seed project for something to start from scratch in angular2 and want to make sure I am not reinventing the wheel: https://github.com/mgechev/angular2-seed

**Comment by amcdnl** [comment]
@zeeshanjan82 Thats a great project, but TBH tho I would NOT use the new router in ng2 right now. 

Check out https://github.com/ui-router/quickstart-ng2 . Neither of those projects use rc1 though so there is some differences you need to account for.
